package.json

```diff
@@ -72,6 +72,7 @@
     "@nodelib/fs.walk": "^1.2.8",
     "ajv": "^6.12.4",
     "chalk": "^4.0.0",
+    "char-source": "^0.0.0",
```

Comment:
Yeah, since you're the author of this package, is there any reason it's just not included in this PR as a utility?

Reply:
Note that that package contains logic that isn't needed in ESLint. The whole validation part for one, including 50% of the unit tests, is not necessary because the syntax of the arguments can be assumed to be valid since it was checked by the parser. There's also additional information about used features (for example, to tell if a string literal is valid in strict mode or not) which seems superfluous unless we expect to use it in ESLint somewhere later.

So if we are going to extract a utility from that package we should probably do some refactoring to remove the unnecessary logic.

Reply:
Yes, let's remove the unnecessary code, of course. As it seems like the package was created just for this purpose (?), I'm assuming that's not an issue.

Reply:
Done in https://github.com/eslint/eslint/pull/18082/commits/53e262a771553b4beeebe60fffed8d16f4fdaaf7, thanks!